src/popover/container.scss

```diff
@@ -56,6 +56,10 @@
   }
 }
 
+.container-body-size-content {
+  // no styles
+}
```

Comment:
1. What is happening here?
2. When using `//` comment (instead of `/**/`), SASS removes them during compilation, so this entire thing is erased. Whatever you wanted to do here, it does not work

Reply:
We add size styles as `container-body-size-${size}` and the style defined the responsiveness of the popover.

For the property filter tokens we want the popover to grow larger and a different approach is used to change layouts for smaller screens.

Reply:
You should use `clsx` library which automatically skip undefined class names. We only render empty test-utils classes, all others should not exist

Reply:
My idea was to add this class as a code documentation in a way, but I can replace it with a code comment. Will update